src/ContentHubClientTrait.php

```diff
@@ -152,9 +152,8 @@ protected function getErrorResponse($code, $reason, $request_id = NULL) {
    *
    *  @codeCoverageIgnore
    */
-  protected function getExceptionResponse($method, array $args, \Exception $exception) {
+  protected function getExceptionResponse($method, string $api_call, \Exception $exception) {
```

Comment:
The method can be more strict, it can only be a string. The return type can be marked and fixed as well.